shell/platform/tizen/flutter_tizen.cc

```diff
@@ -221,9 +221,6 @@ void FlutterNotifyLowMemoryWarning(FlutterWindowControllerRef controller) {
 
 void FlutterRotateWindow(FlutterWindowControllerRef controller,
                          int32_t degree) {
-  if (controller->engine) {
-    controller->engine->SetWindowOrientation(degree);
-  }
 }
```

Comment:
ecore_event_handler_add(ECORE_WL2_EVENT_WINDOW_ROTATE, RotationEventCb, this);
we use this callback instead of system rotate callback, we find system rotate callback will cause app not show issue.
 and I tested  ECORE ROTATE EVENT on mobile simulator, it works same as system rotate and without not show issue.

Reply:
So that means the users cannot control the behavior any more. We can now implement the platform channel's `SystemChrome.setPreferredOrientations` method in order to allow the users to lock the orientation or set preferred orientations. (The implementation can be done later.)

Reply:
SystemChrome.setPreferredOrientations which called from dart, we need implement the platform channel.